src/ui.js

```diff
@@ -36,12 +46,33 @@ export const makeTable = async () => {
   const table = document.createElement("table");
   document.body.appendChild(table); // Drew the main table node on the document
 
-  companiesToDisplay.forEach(row => {
+  companiesToDisplay.forEach((row) => {
     const tr = table.insertRow(); //Create a new row
 
-    row.forEach(column => {
+    row.forEach((column) => {
       const td = tr.insertCell();
       td.innerText = column; // Take string from placeholder variable and append it to <tr> node
     });
   });
-};
\ No newline at end of file
+};
+
+//Converts ISO 8061 time to 24-hour hh:mm format
+function convertTime(dateString) {
```

Comment:
It is a good practice to follow the code style of the other codebase. The rest of codebase uses lambda functions, so here it would be really good to use those, too.